### ISSUE TITLE
Simplify usage of LinVST

### ISSUE THREAD

**Issue by CodingSpiderFox** [osxmidi/LinVst#91]
Many people making music coming from Windows won't have as much of a technical background as us software engineers.

I never used LinVST or anything similar before but I would consider myself as a very experienced Linux user.
When looking at the README of this repo I was really like "no - I don't have time to read through all of this". I just want to make music, don't want to spend hours on setting up my DAW. So the README should be more like:

* starting with a chapter "Getting started" which contains a very quick tutorial
* the setup process explained there not being significantly more complex than:
* download and run this script
* copy your VST dlls etc. to that location
* make these few config settings, start this program

**Comment by osxmidi** [comment]
I added the Quick Start Guide for users that might come from Windows etc and know nothing about Linux.

I tried to explain it as simple as possible, but maybe it can be improved, I'll take another look at it.

The rest of the Readme is just an optional extra that doesn't need to be read at all.

The rest of the Readme is more of a reference for how to deal with some common problems that might come up.

So if someone has trouble getting Waves plugins to run, then they can look that up in the Readme.
